dbt_semantic_interfaces/pyproject.toml

```diff
@@ -24,3 +24,23 @@ more-itertools = "8.10.0"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.black]
```

Comment:
@QMalcolm added this cause there was some CI issues with different black configurations between MF and this repo.